gutenberg.php

```diff
@@ -9,6 +9,11 @@
  * @package gutenberg
  */
 
+require_once dirname( __FILE__ ) . '/lib/check-duplicate-plugins.php';
+if ( defined( 'GUTENBERG_MULTIPLE_COPIES' ) && GUTENBERG_MULTIPLE_COPIES ) {
+	return;
```

Comment:
Our plugin entry point should do as little as possible, only constants and requires and perhaps some very simple checks like in this PR.